resources/packs/gcp/gcp.lr

```diff
@@ -1012,32 +1016,134 @@ private gcp.project.cluster.nodepool.config.kubeletConfig @defaults("cpuManagerP
   podPidsLimit int
 }
 
-// Node pool GCFS configuration
+// GCP GKE node pool GCFS configuration
 private gcp.project.cluster.nodepool.config.gcfsConfig @defaults("enabled") {
   // Internal ID
   id string
   // Whether to use GCFS
   enabled bool
 }
 
-// Advanced machine features configuration
+// GCP GKE node pool advanced machine features configuration
 private gcp.project.cluster.nodepool.config.advancedMachineFeatures @defaults("threadsPerCore") {
   // Internal ID
   id string
   // The number of threads per physical core. If unset, the maximum number of threads supported per core by the underlying processor is assumed
   threadsPerCore int
 }
 
+// GCP GKE node pool GVNIC configuration
 private gcp.project.cluster.nodepool.config.gvnicConfig @defaults("enabled") {
   // Internal ID
   id string
   // Whether to use GVNIC
   enabled bool
 }
 
+// GCP GKE node pool confidential nodes configuration
 private gcp.project.cluster.nodepool.config.confidentialNodes @defaults("enabled") {
   // Internal ID
   id string
   // Whether to use confidential nodes
   enabled bool
-}
\ No newline at end of file
+}
+
+// GCP PubSub resources
+private gcp.project.pubsub {
```

Comment:
This is conflicting with `gcp.project { pubsub() }`. We should rename it to something like `gcp.project.pubsubService`